sopel/modules/currency.py

```diff
@@ -2,109 +2,196 @@
 """
 currency.py - Sopel Currency Conversion Module
 Copyright 2013, Elsie Powell, embolalia.com
+Copyright 2019 Mikkel Jeppesen
```

Comment:
For consistency with (almost) all other module headers:

```suggestion
Copyright 2019, Mikkel Jeppesen
```